gammapy/background/reflected.py

```diff
@@ -89,14 +153,54 @@ def _compute_xy(self, offset, angle):
         y = self.fov['y'] + dy
         return x, y
 
-    def write_off_regions(self, filename):
-        fmt = 'galactic; circle({x},{y},{r})\n'
-        with open(filename, 'w') as fh:
-            for region in self.regions:
-                line = fmt.format(**region)
-                fh.write(line)
 
+def find_reflected_regions_for_event_list(event_list, exclusion,
```

Comment:
I think here a more loose coupling would be good.
Pass in the pointing position as a `SkyCoord` instead of the `EventList`.
Rename to `find_reflected_regions` or `find_reflected_circles`?